core/src/main/scala/org/broadinstitute/dsde/rawls/entities/base/ExpressionValidator.scala

```diff
@@ -58,7 +58,8 @@ trait ExpressionValidator {
     }
   }
 
-  private def validateMCExpressionsInternal(inputs: Map[String, AttributeString],
+  // todo: made public for testing. any other way?
```

Comment:
never resolved this or a similar comment on l. 22 of `LocalEntityExpressionValidator`